eng/packages/http-client-csharp/generator/Azure.Generator/test/common/Azure.Generator.Tests.Common.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <IsTestProject>true</IsTestProject>
+		<IsTestSupportProject>true</IsTestSupportProject>
```

Comment:
this is not a test project, and it is causing error in CI
> Testhost process for source(s) '/mnt/vss/_work/1/s/artifacts/bin/Azure.Generator.Tests.Common/Debug/net8.0/Azure.Generator.Tests.Common.dll' exited with error: Error:
  An assembly specified in the application dependencies manifest (Azure.Generator.Tests.Common.deps.json) was not found:
    package: 'Azure.Core', version: '1.46.1'
    path: 'lib/net8.0/Azure.Core.dll'

https://dev.azure.com/azure-sdk/public/_build/results?buildId=4859942&view=logs&j=5c952eb6-394a-5f6b-81c4-fd6e00460747&t=324a3550-b8a5-548f-4819-7b5944e418be&l=215